### ISSUE TITLE
compose beta - implement input

### ISSUE THREAD

**Issue by JakubDzama2** [vanpra/compose-material-dialogs#32]
In version 0.3.0, in MaterialDialog class, the _input_ composable function is not available

**Comment by vanpra** [comment]
I've had to temporarily remove it due to what seems like an internal bug with compose. I'd have to look more into it to confirm this but I'll add it back as soon as possible.

**Comment by MathewSachin** [comment]
From my observation, the error only happens if the dialog is dismissed when the text field has focus.
I tried releasing focus in the callback and that seems to work. So, releasing focus before dismissing the dialog should fix that.

```
val focusManager = LocalFocusManager.current

// Before dismissing dialog
focusManager.clearFocus(true)
```

**Comment by vanpra** [comment]
Thanks for letting me know @MathewSachin I'll make that change and the `input` function should be back in the next release which will probably be on Sunday.

**Comment by vanpra** [comment]
This should now be fixed in the 0.3.1 release.
